experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/sidebar.html

```diff
@@ -23,4 +23,8 @@
 
     {% endif %}
   {% endfor %}
+  <a class="nav-link d-flex align-items-center" href="#recipe-json">
```

Comment:
Ah no we have to put the whole summary url in here with the anchor tag otherwise clicking it on any of the other pages just does nothing.

Also is there any way clicking it can also expand the hidden JSON? otherwise it just goes to the label but doesn't show anything.